.github/workflows/publish_master_merge.yml

```diff
@@ -0,0 +1,25 @@
+name: master build
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build-and-publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1.3.0
+      with:
+        java-version: 11
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
```

Comment:
curious...can't we have this preset to executable in the repo?

Reply:
good call....looking.